lib/fog/compute/openstack/requests/create_server_group.rb

```diff
@@ -11,6 +11,7 @@ def create_server_group(name, policy)
           }}
           request(
             :body    => Fog::JSON.encode(body),
+            :headers => {'OpenStack-API-Version' => 'compute latest'},
```

Comment:
I'm not sure this is how it is supposed to be done.
* we probably should not set this to 'latest', as we don't support 'future microversions' yet, but rather use a concrete number
* we shouldn't set this header here, but rather for the complete compute client
* maybe this should be done similar to how it was introduced for the shared_file_system client with commit https://github.com/fog/fog-openstack/commit/f72d2fce5fc0c89f9a901eb97767b095f4b3522a#diff-18ad1af87ba332277f23d6e3b943f086 which added a `SUPPORTED_MICROVERSION = '2.15'.freeze`?

Any input, @Carthaca?